src/core/WindowTypes.h

```diff
@@ -1,13 +1,13 @@
 #ifndef _WINDOWTYPES_H_
 #define _WINDOWTYPES_H_
 
-#define WT_CARD QStringLiteral("_WEBOS_WINDOW_TYPE_CARD")
-#define WT_POPUP QStringLiteral("_WEBOS_WINDOW_TYPE_POPUP")
-#define WT_MINIMAL QStringLiteral("_WEBOS_WINDOW_TYPE_RESTRICTED")
-#define WT_OVERLAY QStringLiteral("_WEBOS_WINDOW_TYPE_OVERLAY")
-#define WT_NONE QStringLiteral("_WEBOS_WINDOW_TYPE_NONE")
-#define WT_FLOATING QStringLiteral("_WEBOS_WINDOW_TYPE_FLOATING")
-#define WT_UNKNOWN QStringLiteral("_WEBOS_WINDOW_TYPE_UNKNOWN")
-#define WT_SYSTEM_UI QStringLiteral("_WEBOS_WINDOW_TYPE_SYSTEM_UI")
+#define WT_CARD "_WEBOS_WINDOW_TYPE_CARD"
+#define WT_POPUP "_WEBOS_WINDOW_TYPE_POPUP"
+#define WT_MINIMAL "_WEBOS_WINDOW_TYPE_RESTRICTED"
+#define WT_OVERLAY "_WEBOS_WINDOW_TYPE_OVERLAY"
+#define WT_NONE "_WEBOS_WINDOW_TYPE_NONE"
+#define WT_FLOATING "_WEBOS_WINDOW_TYPE_FLOATING"
+#define WT_UNKNOWN "_WEBOS_WINDOW_TYPE_UNKNOWN"
+#define WT_SYSTEM_UI "_WEBOS_WINDOW_TYPE_SYSTEM_UI"
```

Comment:
(Not mandatory nor needed for this patchset, but...)

We usually prefer const declarations to defines like this. In this case it would be:
const char WT_CARD[]_;

And then in WindowTypes.cpp the actual strings.

Reply:
Done.